docs/src/_guide/advanced-topics.md

```diff
@@ -14,6 +14,39 @@ Most of Meltano's features are available without installing any additional packa
 * `gcs` - Support for using Google Cloud Storage as a [state backend](/concepts/state_backends)
 * `azure` - Support for using Azure Blob Storage as a [state backend](/concepts/state_backends)
 
+## Extension Developer Kit (EDK)
+
+Meltano extensions are lightweight shims that allow you to integrate existing data tools with Meltano.
```

Comment:
Shim can often mean something different. A different word here could be more accurate. Something like...

```suggestion
Meltano extensions are lightweight executables which allow you to integrate existing data tools with Meltano.
```

```suggestion
Meltano extensions are lightweight programs which allow you to integrate existing data tools with Meltano.
```